package.json

```diff
@@ -41,7 +41,7 @@
     "jsonschema": "^1.2.2",
     "moving-average": "^1.0.0",
     "naf-janus-adapter": "^0.10.1",
-    "networked-aframe": "https://github.com/mozillareality/networked-aframe#mr-social-client/master",
+    "networked-aframe": "https://github.com/mozillareality/networked-aframe#bug/aframe-initialization-race-condition",
```

Comment:
can we point back to networked-aframe/master or pull in the latest to the dev fork and keep pointing at that? It got updated with our PR's last week.